proposals/0108-transaction-size-fee.md

```diff
@@ -0,0 +1,190 @@
+---
+simd: '0108'
+title: Transaction Size Fee
+authors:
+  - Jon Cinque (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2024-01-18
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+Add a new transaction fee based on the serialized size of the transaction.
+
+## Motivation
+
+Currently, fees are based entirely on the time it takes to process the
+transaction in a node:
+
+* the fee per signature covers the sigverify time, before execution
+* priority fees are priced on compute units, which cover the virtual machine
+execution time
+
+A transaction takes up more than time resources, however. The network must also
+broadcast the transaction through Turbine so that all nodes can replay it, which
+consumes bandwidth.
```

Comment:
From the perspective of evaluating "resources consumed by transaction", transaction byte size correlates to network ingress/egress cost, and separately long term storage cost. 

I was thinking alone [the line ](https://gist.github.com/tao-stones/6fc77b477d1b946db98e8fe8e3e005ee) that transaction base fee (or its total static fee) accounts for all resources used (including CPUs, memory etc). The cost of bandwidth and ledger storage can all be folded into `CU`. 

(now I caught up with Discord discussion about CU :) I still in fav of pricing everything in CU - perhaps in a different name, that better implies "it takes this amount computing power to do xyz".